embulk-jruby-strptime/src/main/java/org/embulk/spi/time/RubyDateParser.java

```diff
@@ -0,0 +1,112 @@
+package org.embulk.spi.time;
+
+import org.jruby.runtime.ThreadContext;
```

Comment:
I added TODO comments on 1006914f22a1343a60cb76c102a0a6aaeca6dac3. We still need JRuby-dependency tentatively but, it will be removed after merging embulk-jruby-strptime into JRuby.